pac4j-saml/src/main/java/org/pac4j/saml/client/SAML2ClientConfiguration.java

```diff
@@ -19,6 +22,9 @@
  * @since 1.7
  */
 public final class SAML2ClientConfiguration implements Cloneable {
+	private KeyStore keyStore;
+
+	private Resource keystoreResource;
 
     private String keystorePath;
```

Comment:
Do we still really need a `keystorePath`? Cannot it be some encapsulated data of a file resource for the `keystoreResource`?